js/components/LoginScreen.js

```diff
@@ -166,6 +166,15 @@ class LoginScreen extends React.Component {
         />
         <View style={{ height: 10 }} />
         <SpidSubscribeComponent />
+        <View style={{ height: 30 }} />
+        <View style={{ justifyContent: 'center' }}>
```

Comment:
- Use the styles constant.
- We really need a wrapper? Can't we use the `block` property of native-base button?

Reply:
ok, i have use block propertie.